### ISSUE TITLE
a11y - TearOff Homepage - Cognitive Modes Region - SC 4.1.2 - Tabs

### ISSUE THREAD

**Issue by dumaurier** [dumaurier/tearoff_issues#1]
**A custom widget must support the ARIA roles, states, and properties specified by its design pattern.**

- The Cognitive Modes Preview section is missing the requisite tabpanel role 
- (https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/tabpanel_role)
- (https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html)
